# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/kicad/hotfix-camke.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- kicad-10.0.6/cmake/FindwxWidgets.cmake.vanilla	2026-09-05 19:05:27.528688272 +0300
+++ kicad-10.0.6/cmake/FindwxWidgets.cmake	2026-09-05 19:11:05.272401429 +0300
@@ -1029,10 +1029,7 @@
 
 include(FindPackageHandleStandardArgs)
 
-find_package_handle_standard_args(wxWidgets
-  REQUIRED_VARS wxWidgets_LIBRARIES wxWidgets_INCLUDE_DIRS
-  VERSION_VAR   wxWidgets_VERSION_STRING
-  )
+find_package(wxWidgets CONFIG REQUIRED)
 
 #=====================================================================
 # Macros for use in wxWidgets apps.
--- kicad-10.0.6/cmake/Findnng.cmake.vanilla	2026-09-05 19:44:34.348975095 +0300
+++ kicad-10.0.6/cmake/Findnng.cmake	2026-09-05 19:57:41.591781554 +0300
@@ -1,9 +1,5 @@
 find_package(PkgConfig)
 
-if(PKG_CONFIG_FOUND)
-    pkg_check_modules(_NNG nng)
-endif (PKG_CONFIG_FOUND)
-
 FIND_PATH(NNG_INCLUDE_DIR
     NAMES
         nng/nng.h
@@ -21,6 +17,11 @@
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(nng
-        REQUIRED_VARS NNG_INCLUDE_DIR NNG_LIBRARY)
+        NNG_INCLUDE_DIR
+        NNG_LIBRARY
+)
 
 MARK_AS_ADVANCED(NNG_INCLUDE_DIR NNG_LIBRARY)
+
+message(STATUS "DEBUG NNG_INCLUDE_DIR='${NNG_INCLUDE_DIR}'")
+message(STATUS "DEBUG NNG_LIBRARY='${NNG_LIBRARY}'")
--- kicad-10.0.6/CMakeLists.txt.vanilla	2026-09-05 19:13:33.617242666 +0300
+++ kicad-10.0.6/CMakeLists.txt	2026-09-05 19:32:50.437931875 +0300
@@ -1117,12 +1117,6 @@
     message( FATAL_ERROR "Unable to detect wxWidgets port")
 endif()
 
-if( NOT MSVC )
-    if( ${wxWidgets_VERSION_STRING} VERSION_LESS 3.2 )
-        message( FATAL_ERROR "wxWidgets 3.2.0 or greater is required" )
-    endif()
-endif()
-
 if( MINGW )
     # This needs to be on a separate line to protect against a broken FindWxWidgets.cmake in vcpkg
     if( ${wxWidgets_VERSION_STRING} VERSION_LESS 3.1 )

